### PR TITLE
Extension maintenance

### PR DESCRIPTION
**Pull request by kennyadsl**
This PR gives an initial pass to the extension to make it work and have specs passing. It still needs some love since we miss:

- [x] adopt `solidusio-contrib/solidus_extension_dev_tools`
- [x] remove class_eval
- [x] rewrite readme
- [x] run CI on CircleCI
- [x] use Rubocop
- [x] configure coverage
- [ ] refactor specs
- [ ] Admin UI tweaks

### CODE REVIEW

File: app/controllers/spree/admin/locales_controller_decorator.rb [hunk 1]
```diff
@@ -1,15 +1,25 @@
 # frozen_string_literal: true
 
-Spree::Admin::LocaleController.class_eval do
-  before_action :update_i18n_settings, only: :update
+module Spree
+  module Admin
+    module LocalControllerDecorator
```
Comment (aldesantis) [inline app/controllers/spree/admin/locales_controller_decorator.rb]: This should be LocalesControllerDecorator.

**Review by aldesantis — CHANGES_REQUESTED** [review]

File: app/controllers/spree/admin/locales_controller_decorator.rb (hunk 1 above)
Comment (kennyadsl) [reply app/controllers/spree/admin/locales_controller_decorator.rb]: This is probably telling us that this file not needed anymore? 😄

Comment (kennyadsl) [reply app/controllers/spree/admin/locales_controller_decorator.rb]: Nevermind, we are still using it since we prepend it with `self`.

**Review by aldesantis — APPROVED** [review]

**Comment by aldesantis** [comment]
@kennyadsl code looks good. Perhaps the commits could benefit from a reorganization?

**Pull request merged**